lib/config.js

```diff
@@ -408,21 +356,120 @@ exports.get = (cli, callback) => {
+    const LINUX_RESOURCES = path.join(LINUX_OUT_X64, 'resources');
+
+    const LINUX_ICON = CONFIG.src(_.get(PKG, 'config.hadron.build.linux.icon'));
+
+    const LINUX_OUT_DEB = CONFIG.dest(`${CONFIG.slug}-${CONFIG.version}-${CONFIG.arch}.deb`);
+    const LINUX_OUT_RPM = CONFIG.dest(`${CONFIG.slug}.${CONFIG.version}.${CONFIG.arch}.rpm`);
+
+    const LINUX_OUT_TAR = CONFIG.dest(`${CONFIG.slug}-${CONFIG.version}-${CONFIG.platform}-${CONFIG.arch}.tar.gz`);
+    const LINUX_OUT_ZIP = CONFIG.dest(`${CONFIG.slug}.zip`);
 
     _.assign(CONFIG.packagerOptions, {
-      name: LINUX_APPNAME
+      name: CONFIG.productName
     });
 
     CONFIG.resources = LINUX_RESOURCES;
     CONFIG.appPath = LINUX_OUT_X64;
-    CONFIG.assets = [];
+    CONFIG.assets = [
+      {
+        name: `${CONFIG.slug}-${CONFIG.version}-${CONFIG.platform}-${CONFIG.arch}.tar.gz`,
+        path: LINUX_OUT_TAR
+      },
+      {
+        name: `${CONFIG.slug}-${CONFIG.version}-${CONFIG.platform}-${CONFIG.arch}.deb`,
+        path: LINUX_OUT_DEB
+      },
+      {
+        name: `${CONFIG.slug}.${CONFIG.version}.${CONFIG.platform}.${CONFIG.arch}.rpm`,
+        path: LINUX_OUT_RPM
+      },
+      {
+        name: `${CONFIG.slug}-${CONFIG.version}-${CONFIG.platform}-${CONFIG.arch}.zip`,
+        path: LINUX_OUT_ZIP
+      }
+    ];
+
+    const which = require('which');
+    const tarPack = require('tar-pack').pack;
+    const fs = require('fs');
+
+    const createDeb = require('electron-installer-debian');
+    const createRpm = require('electron-installer-redhat');
+
 
     CONFIG.createInstaller = (done) => {
-      cli.warn('Linux installers coming soon!');
-      done();
+      async.parallel([
+        function(cb) {
+          which('rpmbuild', function(err) {
+            if (err) {
+              cli.warn('Your environment is not configured correctly to build ' +
+              'rpm packages. Please see https://git.io/v1iz7');
+              return cb();
+            }
+            /**
+             * TODO (imlucas) Use pretty Redhat metadata and options.
+             * @see https://github.com/unindented/electron-installer-redhat#options
+             */
+            const rpmOptions = {
+              src: LINUX_OUT_X64,
+              dest: CONFIG.out,
+              arch: CONFIG.arch,
+              icon: LINUX_ICON,
+              name: CONFIG.slug,
+              version: CONFIG.version
+            };
+            createRpm(rpmOptions, cb);
+          });
+        },
+        function(cb) {
+          which('fakeroot', function(err) {
+            if (err) {
+              cli.warn('Your environment is not configured correctly to build ' +
+              'debian packages. Please see https://git.io/v1iRV');
+              return cb();
+            }
+            /**
+             * TODO (imlucas) Use pretty debian metadata and options.
+             * @see https://github.com/unindented/electron-installer-debian#options
```

Comment:
`synopsis` generated from description (as our description is also <80 characters, though generally in the same style as a git commit, i.e. read until the `\r\n\r\n` or 79 characters, whichever comes first) would be nice here: https://www.debian.org/doc/debian-policy/ch-binary.html#s-synopsis

Have not looked at any other options though, hopefully lintian helps there.